### PR TITLE
Update microsoft365-apps-usage.md

### PR DESCRIPTION
**Pull request by msbemba**
The data covers up to the past 7 days and we have tested in multiple tenants and it is not 24 to 48 hours. SO made the change to reflect the same in the document.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2683

File: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md [hunk 1]
```diff
@@ -45,11 +45,11 @@ You can get a view into your user's Microsoft 365 Apps activity by looking at th
...
  |8.<br/>|Select the **Manage columns** icon to add or remove columns from the report.|
- |9.<br/>|You can also export the report data into an Excel .csv file by selecting the **Export** link. This exports data for all users and enables you to do simple aggregation, sorting and filtering for further analysis. If you have less than 100 users, you can sort and filter within the table in the report itself. If you have more than 100 users, in order to filter and sort, you will need to export the data.|
\ No newline at end of file
+ |9.<br/>|You can also export the report data into an Excel .csv file by selecting the **Export** link. This exports data for all users and enables you to do simple aggregation, sorting and filtering for further analysis. If you have less than 100 users, you can sort and filter within the table in the report itself. If you have more than 100 users, in order to filter and sort, you will need to export the data.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]:
```suggestion
 |9.<br/>|You can also export the report data into an Excel .csv file by selecting the **Export** link. This exports data for all users and enables you to do simple aggregation, sorting, and filtering for further analysis. If you have less than 100 users, you can sort and filter within the table in the report itself. If you have more than 100 users, in order to filter and sort, you will need to export the data.|
```

File: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md [hunk 2]
```diff
@@ -45,11 +45,11 @@ You can get a view into your user's Microsoft 365 Apps activity by looking at th
 |||
  |:-----|:-----|
  |1. <br/> |The **Microsoft 365 Apps usage** report can be viewed for trends over the last 7 days, 30 days, 90 days, or 180 days. However, if you select a particular day in the report, the table (7) will show data for up to 28 days from the current date (not the date the report was generated). <br/> |
- |2. <br/> |The data in each report usually covers up to the last 24 to 48 hours. <br/> |
+ |2. <br/> |The data in each report usually covers up to the past 7 days. <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]:
```suggestion
 |2. <br/> |The data in each report usually covers up to the last seven days. <br/> |
```

File: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md [hunk 3]
```diff
@@ -45,11 +45,11 @@ You can get a view into your user's Microsoft 365 Apps activity by looking at th
...
  |3. <br/> |The **Users** view iew shows the trend in the number of active users for each app – Outlook, Word, Excel, PowerPoint, OneNote, and Teams. "Active users" are any who perform any intentional actions within these apps. <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]:
```suggestion
 |3. <br/> |The **Users** view shows the trend in the number of active users for each app – Outlook, Word, Excel, PowerPoint, OneNote, and Teams. "Active users" are any who perform any intentional actions within these apps. <br/> |
```

File: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md [hunk 4]
```diff
@@ -45,11 +45,11 @@ You can get a view into your user's Microsoft 365 Apps activity by looking at th
...
  |4. <br/> |The **Platforms** view shows the trend of active users across all apps for each platform – Windows, Mac, Web, and Mobile. <br/> |
  |5.<br/>|On the **Users** chart, the Y axis is the number of unique active users for the respective app. On the **Platforms** chart, the Y axis is the number of unique users for the respective platform. X axis on both charts is the date on which an app was used on a given platform.rm.<br/>|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]:
```suggestion
 |5.<br/>|On the **Users** chart, the Y-axis is the number of unique active users for the respective app. On the **Platforms** chart, the Y-axis is the number of unique users for the respective platform. The X-axis on both charts is the date on which an app was used on a given platform.<br/>|
```

File: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md [hunk 5]
```diff
@@ -45,11 +45,11 @@ You can get a view into your user's Microsoft 365 Apps activity by looking at th
...
  6.<br/>|You can filter the series you see on the chart by selecting an item in the legend. For example, on the **Users** chart, select Outlook, Word, Excel, PowerPoint, OneDrive,or Teams to see only the info related to each one. Changing this selection doesn't change the info in the grid table below it.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]:
```suggestion
 6.<br/>|You can filter the series you see on the chart by selecting an item in the legend. For example, on the **Users** chart, select Outlook, Word, Excel, PowerPoint, OneDrive, or Teams to see only the info related to each one. Changing this selection doesn't change the info in the grid table below it.|
```

File: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md [hunk 6]
```diff
@@ -45,11 +45,11 @@ You can get a view into your user's Microsoft 365 Apps activity by looking at th
...
...which the user activated their Microsoft 365 Apps subscription.<br/><br/>**Last activity date (UTC)** is the latest date an intentional activity was performed by the user. To see activity that occurred on a specific date, select the date directly in the chart.<br/><br/>Following columns corresponding to each app that identifies if the user was active on that app in the period selected:<br> <br>**Outlook** <br>**Word** <br>**Excel**<br>**PowerPoint** <br>**OneNote**<br><br> Following columns corresponding to each platform that identifies if the user was active on that platform for any app (within Microsoft 365 Apps) in the period selected:<br><br>**Outlook (Windows)**<br>**Outlook (Mac)**<br>**Outlook (Web)** <br>**Outlook (Mobile)**<br> **Word (Windows)**<br> **Word (Mac)**<br> **Word (Web)**<br> **Word (Mobile)**<br> **Excel (Windows)**<br> **Excel (Mac)**<br> **Excel (Web)**<br> **Excel (Mobile)**<br> **PowerPoint (Windows)**<br> **PowerPoint (Mac)**<br>**PowerPoint (Web)**<br> **PowerPoint (Mobile)**<br> **OneNote (Windows)**<br> **OneNote (Mac)**<br> **OneNote (Web)**<br>**OneNote (Mobile)**<br> **Teams (Windows)**<br> **Teams (Mac)**<br> **Teams (Web)**<br>**Teams (Mobile)** |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]:
```suggestion
 |7.<br/>|The table shows you a breakdown of data at the per-user level. You can add or remove columns from the table. <br/><br/>**Username** is the email address of the user who performed the activity on Microsoft Apps.<br><br/>**Last activation date (UTC)** is the latest date on which the user activated their Microsoft 365 Apps subscription.<br/><br/>**Last activity date (UTC)** is the latest date an intentional activity was performed by the user. To see activity that occurred on a specific date, select the date directly in the chart.<br/><br/>The following columns correspond to each app that identifies if the user was active on that app in the period selected:<br> <br>**Outlook** <br>**Word** <br>**Excel**<br>**PowerPoint** <br>**OneNote**<br><br> The following columns correspond to each platform that identifies if the user was active on that platform for any app (within Microsoft 365 Apps) in the period selected:<br><br>**Outlook (Windows)**<br>**Outlook (Mac)**<br>**Outlook (Web)** <br>**Outlook (Mobile)**<br> **Word (Windows)**<br> **Word (Mac)**<br> **Word (Web)**<br> **Word (Mobile)**<br> **Excel (Windows)**<br> **Excel (Mac)**<br> **Excel (Web)**<br> **Excel (Mobile)**<br> **PowerPoint (Windows)**<br> **PowerPoint (Mac)**<br>**PowerPoint (Web)**<br> **PowerPoint (Mobile)**<br> **OneNote (Windows)**<br> **OneNote (Mac)**<br> **OneNote (Web)**<br>**OneNote (Mobile)**<br> **Teams (Windows)**<br> **Teams (Mac)**<br> **Teams (Web)**<br>**Teams (Mobile)** |
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/activity-reports/microsoft365-apps-usage.md]: Please change two instances of "Following columns corresponding" to "The following columns correspond".

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba although outside the scope of this PR, please also do the additional edits. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Pull request merged**